examples/SPCTracker.py

```diff
@@ -0,0 +1,158 @@
+"""
+GUI for Storm Prediction Center
+===============================
+
+This GUI takes in SPC data and plots
+all events from a day. It is meant
+to show the ease of data access and
+creation of a useful plot using the
+Siphon Simple Web Service for the SPC.
+"""
+
+
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import ipywidgets as widgets
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+import pandas as pd
+from siphon.simplewebservice import spc
+
+
+class SPC_GUI:
+    """
+    Graphic User Interface designed to allow users to access Storm Prediction Center data.
+
+    This class uses ipython widgets. NOTE: storm chosen must be after 2011 because methods
+    of data parsing are different and will not work for dates earlier.
+    """
+    def __init__(self):
+        """
+        Create object that references SPC.py and thus the Storm Prediction Center.
+
+        This initiation creates the SPC object and also creates a widget that allows
+        the user to select a date for which to animate SPC reports from the years of
+        2011 and onward.
+        """
+        self.datepicker = widgets.DatePicker(description='Pick a Date:', disabled=False)
+        widgets.interact(self.format_date, datepicker=self.datepicker)
+
+    def format_date(self, datepicker):
+        """
+        Allow user to chose a date for which to plot the SPC events.
+
+        Parameters
+        ==========
+        datepicker: ipywidget
+            allows chosing of date
+        """
+        if datepicker is not None:
+            year_string = datepicker.strftime('%Y')
+            month_string = datepicker.strftime('%m')
+            day_string = datepicker.strftime('%d')
+            self.date_string = year_string + month_string + day_string
+            self.stormpicker = widgets.SelectMultiple(options=['tornado', 'hail', 'wind'],
+                                                      description='Event Type: ')
+            widgets.interact(self.fetch_spc, stormpicker=self.stormpicker)
+
+    def fetch_spc(self, stormpicker):
+        """
+        Use a date chosen by the user and create widgets that allow you to iterate through
+        the pandas dataframe holding the data.
+
+        Parameters
+        ==========
+        stormpicker: ipywidget
+            allows choice of the storm type to be plotted
+        """
+        self.spc_data_table = []
+        for storm_type in stormpicker:
+            one_storm_type = spc.SPCD(storm_type, self.date_string)
+            # Storm must be after year 2011
+            if int(self.date_string[:4]) < 2011:
```

Comment:
I'm confused here - are you handling the dates before 2011 by excluding them, because it looks like you are keeping them in here?